keanu-project/src/main/java/io/improbable/keanu/util/status/StatusBar.java

```diff
@@ -109,7 +109,7 @@ private String formatComponents() {
         StringBuilder sb = new StringBuilder();
         for (StatusBarComponent component : components) {
             String render = component.render();
-            if (render != "") {
+            if (!"".equals(render)) {
```

Comment:
2 fixes in one line: 
1. replacing `!=` with `equals()`
2. putting the String literal on the left of the comparison